Documentation/markdown/SetupHelp.md

```diff
@@ -35,7 +37,7 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
 
 ![](./fig/CMake_Installation_settings.png)
 
-7. Choose the options according to [this](./BuildOptions.md)
+7. Choose options according to [this](./BuildOptions.md)
```

Comment:
See above. Also, don't call it *this* :) Write sth of sorts: "For detailed description of options, see [here] ..."

Reply:
Done